bin/lib/book.ml

```diff
@@ -281,28 +295,28 @@ type src = [
 | `Latex
 ]
 
-let make ?(repo_root=".") ~out_dir = function
+let make ?(repo_root=".") ?(include_wip=false) ~out_dir = function
```

Comment:
Is there a reason for this argument to be optional?  I suspect this isn't called in enough places for the concision to be worth making the choice implicit.

Reply:
I made it optional because for the FAQ and the Install pages the argument is unused but I'll make it mandatory!

Reply:
Hmm now that I look at it it feels weird to make it mandatory and to pass a value in `bin/bin/app.ml` knowing it won't be used.

Reply:
I don't really feel strongly. I'm happy with it being released as is.